guide/english/sql/sql-in-operator/index.md

```diff
@@ -49,3 +50,25 @@ where sat_score in (1000, 2400);
 +-----------+------------------------+-----------+---------------------+
 4 rows in set (0.00 sec)
 ```
+
+**Example 2:**
+
+Suppose, we wish to select the details of Customers whose names are one among ['Spencer Pautier', 'Raymond F. Boyce', 'Donald D. Chamberlin', 'Louis Ramsey']. Instead of writing four equality conditions separated by `OR` we could use the `IN` operator.
+
+```sql
+select studentID, FullName, sat_score, rcd_updated 
+from student 
+where FullName in ('Spencer Pautier', 'Raymond F. Boyce', 'Louis Ramsey', 'Donald D. Chamberlin');
```

Comment:
in => IN ... ?

Reply:
Yes. Since the author of the article used lowercase for all sql statements, I followed suit.

Reply:
Sorry missed this one.